test/pom.xml

```diff
@@ -127,10 +127,6 @@
       <groupId>org.apache.accumulo</groupId>
       <artifactId>accumulo-tserver</artifactId>
     </dependency>
-    <dependency>
```

Comment:
@rsingh433  You removed the duplicate dependency entry on line 90, but also removed this original entry. 
Once you restore this entry, I believe your build should pass.